zeroex/ordervalidator/order_validator.go

```diff
@@ -837,23 +821,16 @@ const emptyGetOrderRelevantStatesCallDataByteLength = 268
 */
 const jsonRPCPayloadByteLength = 444
 
-func (o *OrderValidator) computeABIEncodedSignedOrderByteLength(signedOrder *zeroex.SignedOrder) (int, error) {
-	trimmedOrder := signedOrder.Trim()
-	data, err := o.devUtilsABI.Pack(
-		"getOrderRelevantStates",
-		[]wrappers.LibOrderOrder{trimmedOrder},
-		[][]byte{signedOrder.Signature},
-	)
-	if err != nil {
-		return 0, err
-	}
-	dataBytes := hexutil.Bytes(data)
-	encodedData, err := json.Marshal(dataBytes)
-	if err != nil {
-		return 0, err
-	}
-	encodedSignedOrderByteLength := len(encodedData) - emptyGetOrderRelevantStatesCallDataByteLength
-	return encodedSignedOrderByteLength, nil
+func numWords(bytes []byte) int {
+	return (len(bytes) + 31) / 32
+}
+
+func computeABIEncodedSignedOrderByteLength(signedOrder *zeroex.SignedOrder) int {
+	// The fixed size fields in a SignedOrder take up 1536 bytes. The variable length fields take up 64 bytes per 256-bit word.
```

Comment:
Where did you get that the variable length fields take up 64 bytes per 256 bit word? 256 bits is 32 bytes, and ABI encoding should only include an extra offset and length slot. I could be misunderstanding this, but this is fairly confusing to me without more context.

Reply:
it takes up 64 bytes because in JSON each byte requires two characters. I'll add that to the comment.